assimilation_code/modules/utilities/cray_win_mod.f90

```diff
@@ -54,9 +54,11 @@ module window_mod
 !-------------------------------------------------------------
 !> Create the window for the ensemble complete state vector
 !> I think you have to pass it the state ensemble handle
-subroutine create_state_window(state_ens_handle)
+subroutine create_state_window(state_ens_handle, fwd_op_ens_handle, qc_ens_handle)
```

Comment:
This is a question for my own understanding, 
the fwd and qc arguments are optional because there are cases where create_state_window is called where there is no fwd_op or qc handle? e.g. model_mod_check
https://github.com/NCAR/DART/blob/16595cad595b6349605c9bc8e021c7ecc2c2391a/assimilation_code/programs/model_mod_check/model_mod_check.f90#L250

Reply:
Yep, I made them optional because I saw that there were some calls to create_state_window, such as the one in model_mod_check above, that only uses the ens_handle argument. Does this sound good to you?

Reply:
Yes definitely